cfgov/v1/__init__.py

```diff
@@ -93,6 +93,8 @@ def parse_links(soup):
 def external_links_filter(value):
     if isinstance(value, RichText):
         return parse_links(BeautifulSoup(expand_db_html(value.source), 'html.parser'))
+    else:
+        return parse_links(BeautifulSoup(expand_db_html(value), 'html.parser'))
     return value
```

Comment:
`return value` would never be hit here. are we sure that there isn't any other condition that could be met than the two RichText conditions?